static/js/libs/fileValidation.js

```diff
@@ -0,0 +1,106 @@
+function baseRestrictions(file, restrictions) {
+  return new Promise((resolve, reject) => {
+    const errors = [];
+    if (restrictions.accept) {
+      if (restrictions.accept.indexOf(file.type) === -1) {
+        errors.push("File type is incorrect");
+      }
+    }
+
+    if (restrictions.size) {
+      const fileSize = file.size / 1000000;
```

Comment:
Wouldn't it be easier to make the restriction value in bytes to avoid floating point comparisons?

Reply:
yeah, it's just not very friendly - but I guess they won't change that often